packages/server/package.json

```diff
@@ -58,30 +58,37 @@
         "@contentful/rich-text-from-markdown": "^15.16.15",
         "@oclif/core": "^3.26.4",
         "@types/lodash": "^4.14.202",
+        "@types/pdf-parse": "^1.1.4",
         "@types/uuid": "^9.0.7",
         "async-mutex": "^0.4.0",
         "axios": "1.6.2",
         "content-disposition": "0.5.4",
         "contentful-management": "^11.7.3",
         "cors": "^2.8.5",
         "crypto-js": "^4.1.1",
-        "langchainhub": "^0.0.11",
         "dotenv": "^16.0.0",
         "express": "^4.17.3",
         "express-basic-auth": "^1.2.1",
         "express-oauth2-jwt-bearer": "^1.6.0",
         "express-rate-limit": "^6.9.0",
+        "express-session": "^1.18.1",
         "flowise-components": "workspace:^",
         "flowise-ui": "workspace:^",
+        "google-auth-library": "^9.4.0",
+        "googleapis": "^144.0.0",
         "http-errors": "^2.0.0",
         "http-status-codes": "^2.3.0",
         "langchainhub": "^0.0.11",
         "lodash": "^4.17.21",
+        "mammoth": "^1.9.0",
         "moment": "^2.29.3",
         "moment-timezone": "^0.5.34",
         "multer": "^1.4.5-lts.1",
         "mysql2": "^3.9.2",
         "openai": "^4.51.0",
+        "passport": "^0.7.0",
+        "passport-google-oauth20": "^2.0.0",
+        "pdf-parse": "^1.1.1",
```

Comment:
Why is this one needed? Do we need this pdf-parse? We should be using the existing PDF support.